miden/tests/integration/operations/ext2_ops.rs

```diff
@@ -0,0 +1,121 @@
+use crate::build_op_test;
+use rand_utils::rand_value;
+use vm_core::{Felt, FieldElement, QuadExtension, StarkField};
+
+type ExtElement = QuadExtension<Felt>;
+
+// EXT2 OPS ASSERTIONS - MANUAL TESTS
+// ================================================================================================
+
+#[test]
+fn ext2add() {
+    let asm_op = "ext2add";
+
+    let a0 = Felt::new(rand_value::<u64>());
+    let a1 = Felt::new(rand_value::<u64>());
+    let b0 = Felt::new(rand_value::<u64>());
+    let b1 = Felt::new(rand_value::<u64>());
+
+    let a = ExtElement::new(a0, a1);
+    let b = ExtElement::new(b0, b1);
```

Comment:
You should be able to do something like:
```Rust
let a = rand_value::<ExtElement>();
let b = rand_value::<ExtElement>();
```
But not sure if this will simplify things or not.